### ISSUE TITLE
How to use two different comma type?

### ISSUE THREAD

**Issue by odg0318** [gocarina/gocsv#84]
Hello.
I am trying to parse two csv files that have different comma type; ",", "\t", in a program.
I injected below code to change default separator.
```golang
        gocsv.SetCSVReader(func(in io.Reader) gocsv.CSVReader {
                r := csv.NewReader(in)
                r.Comma = '\t'
                return r
        })
```
It seems like working fine but doesn't.
The csv with tab could be parsed well but at the same time one with comma not.
In my opinion, the same reader is used in a program.

How can I use different reader on each csv file?

**Comment by pikanezi** [comment]
Hello,

What you're doing setting the Comma to '\t' then you're setting it to ',' and it does not work, have I understood it right?

**Comment by odg0318** [comment]
In my program, two kinds of csv files using comma or tab are parsed at the same time.
But only one reader can be set via your library and so it is impossible tha gocsv parses both of them.

**Comment by SmaugTheGreat** [comment]
Yes, this is also a problem that I have with this library :( Unfortunately when using two different readers in 2 goroutines it won't work because it's global.

**Comment by vk18vk** [comment]
did anyone found how to change comma type while using this library

**Comment by Destructive-Dragon** [comment]
The problem is the usage of global variables which is generally bad coding style. Someone should make a pull request to fix this.
